pkg/cmd/server/origin/master.go

```diff
@@ -436,7 +436,8 @@ func (c *MasterConfig) GetRestStorage() map[string]rest.Storage {
 		"imageStreamMappings": imageStreamMappingStorage,
 		"imageStreamTags":     imageStreamTagStorage,
 
-		"deploymentConfigs":         deployConfigStorage,
+		"deploymentConfigs":         deployConfigStorage.DeploymentConfig,
+		"deploymentConfigs/scale":   deployConfigStorage.Scale,
```

Comment:
need to update policy with any new resources (either resource group definitions in `authorization/api/types.go` or bootstrap roles in `cmd/server/bootstrappolicy/policy.go` or both)